CRM/Member/Page/RecurringContributions.php

```diff
@@ -112,14 +112,18 @@ private function getRecurContributions($membershipID) {
    */
   private function setActionsForRecurringContribution($recurID, &$recurringContribution) {
     $action = array_sum(array_keys($this->recurLinks($recurID)));
+
     // no action allowed if it's not active
     $recurringContribution['is_active'] = ($recurringContribution['contribution_status_id'] != 3);
+
     if ($recurringContribution['is_active']) {
       $details = CRM_Contribute_BAO_ContributionRecur::getSubscriptionDetails($recurringContribution['id'], 'recur');
       $hideUpdate = $details->membership_id & $details->auto_renew;
-      if ($hideUpdate || empty($details->processor_id)) {

```

Comment:
@MiyaNoctem I've just tested editing with no payment processor linked to the recurring and that is fine because you get the message "There is no valid processor for this subscription so it cannot be edited."  Have you tested with a payment processor that does not support editing recurring contributions?

Reply:
@mattwire Haven't tried it, but it should work exactly as it works on the contributions tab, that part remains unchanged.